imapclient/livetest.py

```diff
@@ -918,6 +918,26 @@ def test_expunge(self):
                 # on. EXPUNGE won't return anything in this case
                 self.assertIn((1, b'EXPUNGE'), resps)
 
+        def test_id_expunge(self):
+          if not self.client.use_uid:
+            self.skipTest('test instance not configured for UID operations')
+          folder = self.add_prefix_to_folder('test_id_expunge')
+          self.client.create_folder(folder)
+          self.client.select_folder(folder)
+          for i in range(3):
+            self.client.append(folder, 'Subject: msg %d\r\n\r\nbody %d\r\n\r\n' % (i, i))
+          messages = self.client.search()
+          self.assertEquals(len(messages), 3)
```

Comment:
`assertEquals` is a deprecated alias of `assertEqual` (without final s). The others test are using the latter, so it would be better without the extra `s` :) 

And same thing than NicolasLM: four spaces indent on this file as well.

Reply:
4 space indent and `assertEquals` -> `assertEqual` done, rev# b566947e46dd5699f66c12507e383f7578f36942